File: backend/BlockChain.Domain/BlockChain/Transacao.cs
using BlockChain.Cross.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BlockChain.Domain.BlockChain
{
    
    public class Transacao : Entity<Guid>
    {
        
        public DateTime DataTrasacao { get; set; }        
        public string TipoTransacao { get; set; }
        public string CodigoTransacao { get; set; }

        public string CodigoCarteiraOrigem { get; set; }
        public string CodigoCarteiraDestino { get; set; }
        public float Saldo { get; set; }
        
    }
}

File: frontend/src/app/model/Transacao.ts
import { BaseEntity } from './BaseEntity';

/*import { TipocomandOperacao } from './TipocomandOperacao';
import { TipoMotivorestricao } from './TipoMotivorestricao';
import { ComandOperacao } from './ComandOperacao';*/


export class Transacao extends BaseEntity<number> {


    public TipoTransacao?: string;

    public CodigoTransacao?: string;
    
    public CodigoCarteiraOrigem?: string;

    public CodigoCarteiraDestino?: string;

    public DataTrasacao?: string;    

    public Saldo?: number;   
    

}